Makefile

```diff
@@ -413,14 +460,17 @@ test-e2e-local-migration: $(GINKGO)
 test-e2e-local: $(GINKGO)
 	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter="default"
 
+test-e2e-local-ha: $(GINKGO)
+	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter "simple || (high-availability && upgrade-to-$(SHOOT_FAILURE_TOLERANCE_TYPE))"
+
 ci-e2e-kind: $(KIND) $(YQ)
 	./hack/ci-e2e-kind.sh
 
 ci-e2e-kind-ha-node: $(KIND) $(YQ)
-	SHOOT_FAILURE_TOLERANCE_TYPE=node ./hack/ci-e2e-kind-ha.sh
+	SHOOT_FAILURE_TOLERANCE_TYPE=node ./hack/ci-e2e-kind-ha-node.sh
 
 ci-e2e-kind-ha-zone: $(KIND) $(YQ)
-	SHOOT_FAILURE_TOLERANCE_TYPE=zone ./hack/ci-e2e-kind-ha.sh
+	SHOOT_FAILURE_TOLERANCE_TYPE=zone ./hack/ci-e2e-kind-ha-zone.sh
```

Comment:
Now that we have dedicated files, can we remove `SHOOT_FAILURE_TOLERANCE_TYPE` from here?

Reply:
Based on `SHOOT_FAILURE_TOLERANCE_TYPE ` we will create a shoot with corresponding failureToleranceType  [here](https://github.com/gardener/gardener/blob/master/test/e2e/common.go#L43-L59). So we need this env `SHOOT_FAILURE_TOLERANCE_TYPE`.